pull.py

```diff
@@ -80,15 +106,60 @@ def main():
         stocks.loc['Change'] = stocks.iloc[0] - stocks.iloc[-1]
         print(stocks.loc['Change'])
 
-    py.plot({
-        "data": [go.Scatter(
-            x=stocks.index,
-            y=stocks[col],
-            name=col)
-            for col in stocks.columns],
-        "layout": go.Layout(title="Daily Openings")
-    }, auto_open=True)
+    if args.plot_line:
+        stocks = stocks.transpose()
```

Comment:
What I would recommend is creating fuctions for each of these different plotting methods. That will keep the code cleaner and allow you organize things in a better, testable fashion.